plugins/package-managers/cocoapods/src/main/kotlin/CocoaPods.kt

```diff
@@ -251,14 +257,18 @@ private fun getPackageReferences(podfileLock: File): Set<PackageReference> {
         val (name, version) = parseNameAndVersion(entry)
         versionForName[name] = checkNotNull(version)
 
-        val dependencies = node[entry]?.map { it.textValue().substringBefore(" ") }.orEmpty()
+        val dependencies = node[entry]?.map { depNode ->
```

Comment:
Is this `depNode` of same format as `node`? e.g. would the case distinction from line 252 through 255 need to be applied as well?

Reply:
No, `depNode` is always a text node (as nesting can be max. 2 levels deep in the `PODS` section).